### ISSUE TITLE
Warning: Calling bottle :unneeded is deprecated! There is no replacement.

### ISSUE THREAD

**Issue by 9mm** [knqyf263/pet#182]
```
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the knqyf263/pet tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/knqyf263/homebrew-pet/pet.rb:6
```

Just started getting this in homebrew

**Comment by JosephKu** [comment]
Please update [GorReleaser](https://github.com/goreleaser/goreleaser/pull/2591) and [knqyf263/homebrew-pet](https://github.com/knqyf263/homebrew-pet) to remove this warning. Thanks!

**Comment by 9mm** [comment]
@knqyf263 any chance we can fix this?

**Comment by knqyf263** [comment]
I need someone's help.

**Comment by JosephKu** [comment]
@knqyf263 What kind of help do you need? I could help to fix it. However, the repo [knqyf263/homebrew-pet](https://github.com/knqyf263/homebrew-pet) still needed to be updated for Homebrew.

**Comment by knqyf263** [comment]
Thanks. We have to update the GoReleaser version which addresses the warning, but [this config file](https://github.com/knqyf263/pet/blob/main/.goreleaser.yml) is too stale to work with the latest GoReleaser. Once all errors are resolved, we can release the next version and it would fix the warning.

**Comment by JosephKu** [comment]
@knqyf263 #183 Is this what you need?

**Comment by knqyf263** [comment]
Thanks. Let me check.

**Comment by knqyf263** [comment]
Thank you all! I've released v0.4.0. Hopefully, it addresses this issue.

**Comment by 9mm** [comment]
works, thanks so much guys!
